CHANGELOG/CHANGELOG-0.8.md

```diff
@@ -0,0 +1,19 @@
+# Changelog 0.8
+
+## 0.8.0 - 2023-10-26
+
+### Changed
+
+- Updated the Grafana docker image for future plugins. ([#524](https://github.com/antrea-io/theia/pull/524), [@Dhruv-J])
+- Updated the node version and toolkit to create plugin ([#536](https://github.com/antrea-io/theia/pull/536), [@yuntanghsu])
```

Comment:
s/and toolkit to create plugin/and replaced toolkit with create-plugin for plugin development